common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -347,19 +350,30 @@ public boolean close() {
         @Override
         public String toString() {
             if (creationRecord == null) {
-                return "";
+                return StringUtil.EMPTY_STRING;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'StringUtil.EMPTY_STRING' due to existing static import 'io.netty.util.internal.StringUtil.*' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)